.dockerignore

```diff
@@ -0,0 +1,30 @@
+build/
+TODO
+circleci-cli
+coverage.txt
+docs/
+out/
+vendor/
+.vscode
+
+# For supporting generated config of 2.1 syntax for local testing
+.circleci/processed.yml
+
+# Snapcraft Related
+parts/
+prime/
+snap/.snapcraft/
+stage/
+*.snap
+
+# packr related
+*/*-packr.go
+packrd/
+
+# golangci-lint
+# We expect to install the binary each time in CI
+bin/golangci-lint
+integration_tests/tmp
+*.exe
+
+Dockerfile*
```

Comment:
Just curious, does this file improve the user experience of the image at all, or does it just optimize the speed of `docker build`?